File: src/main/java/life/magpies/registry/commands/ModCommands.java
package life.magpies.registry.commands;

import life.magpies.SimplerTemplateMod;

public class ModCommands {
    public static void registerCommands(){
        CommandKit.register();
        CommandDay.register();
        CommandWeather.register();
        CommandList.register();
        CommandTpa.register();
        CommandTpa.register();
        CommandTpAccept.register();
        CommandTpc.register();
        CommandTpcAetp.register();
    }
    public static void registerModCommand() {
        SimplerTemplateMod.LOGGER.debug("Registering mod command for" + SimplerTemplateMod.MOD_ID);
    }
}

File: src/main/kotlin/life/magpies/registry/commands/ModCommands.kt
package life.magpies.registry.commands

object ModCommands {
    fun registerCommands() {
        CommandWeather.register()
        CommandKit.register()
        CommandThis.register()
        CommandDay.register()
        CommandTpa.register()
        CommandTpAccept.register()
        CommandTpc.register()
        CommandTpcAetp.register()
        CommandList.register()
    }
}